integration/test/basic/basic_test.go

```diff
@@ -27,39 +32,55 @@ func TestChartInstalled(t *testing.T) {
 		RestConfig:      f.RestConfig(),
 		TillerNamespace: "giantswarm",
 	}
-	helmClient, err := helmclient.New(c)
+
+	gsHelmClient, err := helmclient.New(c)
 	if err != nil {
 		t.Fatalf("could not create helmClient %v", err)
 	}
 
-	err = f.InstallResource("chart-operator-resource", templates.ChartOperatorResourceValues, ":stable")
+	// Test Creation
+	log.Printf("creating %q", cr)
+	err = f.InstallResource(cr, templates.ChartOperatorResourceValues, ":stable")
+	if err != nil {
+		t.Fatalf("could not install %q %v", cr, err)
+	}
+
+	err = waitForReleaseStatus(gsHelmClient, testRelease, "DEPLOYED")
+	if err != nil {
+		t.Fatalf("could not get release status of %q %v", testRelease, err)
+	}
+	log.Printf("%q succesfully deployed", testRelease)
+
+	// Test Deletion
+	log.Printf("deleting %q", cr)
+	err = helmClient.DeleteRelease(cr)
 	if err != nil {
-		t.Fatalf("could not install chart-operator-resource-chart %v", err)
+		t.Fatalf("could not delete %q %v", cr, err)
 	}
 
-	var rc *helmclient.ReleaseContent
+	err = waitForReleaseStatus(gsHelmClient, testRelease, "DELETED")
+	if !helmclient.IsReleaseNotFound(err) {
+		t.Fatalf("%q not succesfully deleted %v", testRelease, err)
+	}
+	log.Printf("%q succesfully deleted", testRelease)
+}
+
+func waitForReleaseStatus(gsHelmClient *helmclient.Client, release string, status string) error {
 	operation := func() error {
-		rc, err = helmClient.GetReleaseContent("tb-release")
+		rc, err := gsHelmClient.GetReleaseContent(release)
```

Comment:
Refactor Status checks to prepare for `Update` test. Note that the `gsHelmClient` is used here since guest services will be installed with our own tiller.